util/password/password.go

```diff
@@ -70,3 +83,37 @@ func GetAccountPassword() ([]byte, error) {
 	}
 	return []byte(Passwd), nil
 }
+
+func ReadPasswordFile() ([]byte, error) {
+	var file []byte
+
+	passwordFile := config.Parameters.PasswordFile
+
+	_, err := os.Stat(passwordFile)
+	if err != nil {
+		return file, err
+	}
+
+	file, err = ioutil.ReadFile(passwordFile)
+	if err != nil {
+		return nil, err
+	}
+
+	// Remove the UTF-8 Byte Order Mark
+	file = bytes.TrimPrefix(file, []byte("\xef\xbb\xbf"))
+	return file, err
+}
+
+func SavePassword() error {
```

Comment:
It's better to pass password to this function because Passwd is not always guaranteed to contain password (unless passed in through -p argument or created in dashboard)